### PR TITLE
Support GLPK 5.0 and update build process

### PR DESCRIPTION
**Pull request by cdiener**
This enables support for GLPK 5.0+ by removing functions with missing symbols in the shared library. Also updates the supported Python versions to 3.6-3.9 and supplied SWIG to 4.0.2.

### CODE REVIEW

**Comment by cdiener** [comment]
Hmm, travis is not triggered. Dunno what's going on here. Might want to change to GH Actions at some point.

File: setup.py [hunk 1]
```diff
@@ -19,13 +19,24 @@
...
+    r'^.+netgen.+\n*.+;',
+    r'^.+gridgen.+\n*.+;',
+]
+
+
 def copy_glpk_header():
     if os.path.isfile('glpk.h'):
         print('glpk.h found in source directory')
         glpk_header_path = os.path.join('./', 'glpk.h')
+    elif "GLPK_HEADER_PATH" in os.environ:
```
Comment (cdiener) [inline setup.py:38]: This should make it easier to build on conda-forge in the future :crossed_fingers:

**Comment by phantomas1234** [comment]
@cdiener you think it would be useful to move this one to GH actions (have you figured out why travis is not triggered)? @carrascomj just moved cameo to GH actions, maybe he could do this one too?

**Review by phantomas1234 — APPROVED** [review]
LGTM

**Comment by cdiener** [comment]
GH actions for linux and mac wouldn't be too hard with https://github.com/joerick/cibuildwheel . I would probably hold of with Windows builds for now because it works on appveyor and there might be some caveats. If the new wheels for Python 3.9 are out this could be tackled in a separate PR.

I'll close this PR in favour of the other.

**Pull request closed without merging**